server/views/pages/timetable.njk

```diff
@@ -8,18 +8,28 @@
 
 {% set backLinkHref = "/" %}
 
-{% macro attendees(enhanced, prisonerLocationGroupNames) %}
+{% macro attendees(enhanced, prisonerCategoryGroupNames, prisonerLocationGroupNames) %}
```

Comment:
Trying to wedge several different bits of data in to one table cell here so it's not particularly elegant. I think the design is being reconsidered so hopefully can be simplified in future!